package.json

```diff
@@ -73,8 +78,8 @@
     "gemini-scrollbar": "github:matrix-org/gemini-scrollbar#b302279",
     "gfm.css": "^1.1.2",
     "highlight.js": "^9.13.1",
-    "matrix-js-sdk": "1.0.4",
-    "matrix-react-sdk": "1.0.7",
+    "matrix-js-sdk": "2.2.0",
```

Comment:
:thinking: ça va utiliser des versions différentes du JS sdk sur react-sdk et ici ?

Reply:
faut faire gaffe de les garder in sync, oui... Je sais pas dans le detail pourquoi ya un import ici aussi.

Reply:
Il y a une faille de sécurité dans le js-sdk qui a été corrigé dans la version patché de Leo. Ce n'est pas le cas dans la 2.2.0 je pense

Reply:
On peut merge et régler ce problème avant mise en beta